### PR TITLE
new migration guide for integrators and exchanges

### PR DESCRIPTION
**Pull request by ines-toupeira**

### CODE REVIEW

File: migration/migration-integrators.md [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+7. [`createBasicTransaction`](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#createbasictransaction): Create a signed transaction and returns a hex-encoded representation without broadcasting it. All `create*Transaction` RPC methods also have an equivalent `send*Transaction` version in order to create and broadcast the transaction one-go.
+8. [`sendRawTransaction`](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#sendrawtransaction): Sends the given serialized and signed transaction to the network.
+
+### Shutdown of PoW Nodes
+
+Once the PoS chain has started, PoW nodes are no longer needed and can be shut down. This also acts as a commitment to leave the PoW chain behind. Make sure that accessing transactions on the PoW chain is no longer necessary once the PoS Mainnet chain is live. **If you need to access transaction history from before the transition, you will need to complete the migration process as described in the [Migration Tool Guide](node-operators.md).**
+
+## Getting Started with the PoS Bode
```
Comment (Eligioo) [inline migration/migration-integrators.md]:
```suggestion
## Getting Started with the PoS Node
```

File: migration/migration-integrators.md [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+- **network.tls:** TLS is **optional**; the client uses the [libp2p](https://libp2p.io) network stack under the hood which enforces encrypted connections between peers out-of-the-box.
+- **consensus.network**: Set to `main-albatross` in order to connect to the PoS Mainnet (default is Testnet).
+- **consensus.sync_mode**: **Must** be `history` or `full` as outlined in [General Considerations](#general-considerations-for-different-node-options).
+- **consensus.max_epochs_stored:** Only applies if `consensus.sync_mode` is set to `full`; configures how long blocks and transactions are kept around before they get pruned (1 epoch translates roughly to 12 hours).
+- **rpc-server**: Uncomment the JSON-RPC server section (which it is by default) and configure the bind address and port for remote communication if desired. However, it is not recommended to allow communication outside of localhost unless TLS is used (please verify if TLS support is enabled in your setup).
```
Comment (Eligioo) [inline migration/migration-integrators.md]:
```suggestion
- **rpc-server**: Uncomment the JSON-RPC server section (which it is by default) and configure the bind address and port for remote communication if desired. However, it is not recommended to allow communication outside of localhost unless TLS is used.
```

**Review by Eligioo — APPROVED** [review]
LGTM, 2 comments

**Comment by onmax** [comment]
deployed

**Pull request merged**